azkaban-web-server/build.gradle

```diff
@@ -15,7 +15,8 @@ buildscript {
 // manage NodeJS distributions, use them from there.
 // ***npm install*** installs all dependencies in package.json. It will only run when changes are made to package.json
 plugins {
-    id "com.moowork.node" version "1.2.0"
+//    id "com.moowork.node" version "1.2.0"
+    id("com.github.node-gradle.node") version "2.2.0"
```

Comment:
why do we need bump the node version here? It is for web server, not revelant to k8s pod side if I understand correctly

Reply:
I cannot build this module on my side without it - always get this error on node setup:

```
Execution failed for task ':azkaban-web-server:nodeSetup'.
> Could not resolve all files for configuration ':azkaban-web-server:detachedConfiguration1'.
   > Could not resolve org.nodejs:node:8.10.0.
     Required by:
         project :azkaban-web-server
      > Could not resolve org.nodejs:node:8.10.0.
         > Could not get resource 'https://nodejs.org/dist/v8.10.0/ivy.xml'.
            > Could not GET 'https://nodejs.org/dist/v8.10.0/ivy.xml'. Received status code 403 from server: Forbidden
```

Are you able to build it locally? If so then I believe the issue is only on me and I can revert the changes

Reply:
I can build it locally but I guess it is only because I have that cache copy, the dependency haven't been built for a long time. 
I might encourage to resolve this issue in another PR. As it is reflecting for web server (not directly impact/necessary in your change ) and I do see the entire package changed too, so we don't really know the real impact for that. I would be conservative for any dependency change.

Reply:
sure, I have reverted this change. Also made my local build work too without this change (disabling the download locally).